encoder.py

```diff
@@ -108,9 +100,9 @@ def _get_features(self, content):
         return self.model(content)
 
     def _get_pooling(self, feature_map: 'np.ndarray') -> 'np.ndarray':
```

Comment:
This is not going to work on `GPU` right?

Also I am not sure why we need to take care of this? the first condition, does it ever happen? 

I think the pooling should be done in the torch tensor directly that may be in GPU as the layer

Reply:
I removed the if statement in https://github.com/jina-ai/executor-image-torch-encoder/pull/7/commits/4a4775e667e1787f03b34e74bb7659f3d65628c9

I would expect it to work on GPU now, but I can't test it locally. 
There is a unit tests for GPU usage, if you have one, could you try?